common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1846,6 +1846,29 @@
     },
     "sqlState" : "42000"
   },
+  "INVALID_INVERSE_DISTRIBUTION_FUNCTION" : {
+    "message" : [
+      "Invalid inverse distribution function <funcName>."
+    ],
+    "subClass" : {
+      "DISTINCT_UNSUPPORTED" : {
+        "message" : [
+          "Cannot use DISTINCT with WITHIN GROUP."
+        ]
+      },
+      "PERCENTILE_PERCENTAGE_MISSING" : {
+        "message" : [
+          "Missing percentage."
```

Comment:
This doesn't seem like a thing to inverse distribution function. It should be specific to certain functions and shouldn't be in this error class.